packetbeat/scripts/mage/pcap.go

```diff
@@ -18,23 +18,14 @@
 package mage
 
 import (
-	"log"
-	"strings"
-
 	devtools "github.com/elastic/beats/v7/dev-tools/mage"
 
-	"github.com/magefile/mage/mg"
-	"github.com/magefile/mage/sh"
 	"github.com/pkg/errors"
```

Comment:
please run goimports over your file. The CI is complaining because this import is not used (unused symbols are illegal in Go).